src/janet.zig

```diff
@@ -2291,3 +2291,17 @@ test "function call" {
     try func.pcall(&[_]Janet{ wrap(i32, 2), wrap(i32, 2) }, &sum, null);
     try testing.expectEqual(@as(i32, 4), try sum.unwrap(i32));
 }
+
+test "os/execute" {
+    if (@import("builtin").target.os.tag == .linux) {
+        try init();
+        defer deinit();
+        const env = Environment.coreEnv(null);
+        // todo: maybe spork/sh is not installed
+        const value = try env.doString(
+            \\(import spork/sh) (sh/exec-slurp "uname")
+        , "main");
+        const s = try value.unwrap(String); // it should be string
+        try testing.expectEqualStrings("Linux", std.mem.span(@as([*:0]const u8, @ptrCast(s.slice.ptr))));
+    }
+}
```

Comment:
This looks like a bad test? Only for Linux and only with spork installed. I don't think it is good for newcomers, it increases the barrier to enter unnecessarily. We could probably take the most necessary part from `sh/exec-slurp` and use it here verbatim.

I also wouldn't mind deleting it altogether since this is not a bug for this library but rather upstream. I'm not sure whether this test tests anything from these bindings or rather a general test that belongs to Janet itself.

Reply:
good point. I'll delete the test